pkg/generate/git/repository.go

```diff
@@ -364,7 +364,7 @@ func command(name, dir string, env []string, args ...string) (stdout, stderr str
 func timedCommand(timeout time.Duration, name, dir string, env []string, args ...string) (stdout, stderr string, err error) {
 	var stdoutBuffer, stderrBuffer bytes.Buffer
 
-	glog.V(4).Infof("Executing %s %s %s", strings.Join(env, " "), name, strings.Join(args, " "))
+	glog.V(4).Infof("Executing %s %s", name, strings.Join(args, " "))
```

Comment:
In general logs should contain no more security info than necessary, and ideally none (just enough to identify that you passed the right secret).  Reprinting the env hundreds of times makes the logs not useful.